lib/report.js

```diff
@@ -1,10 +1,10 @@
 const Exclude = require('test-exclude')
-const furi = require('furi')
```

Comment:
can we remove this from the `package.json` as well?

Reply:
We can indeed